crates/release_plz/src/args/shell_completions.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::anyhow;
+use clap::IntoApp;
+use clap_complete::Shell;
+use std::str::FromStr;
+
+#[derive(clap::Parser, Debug)]
+pub struct ShellCompletions {
+    /// Shell option
+    #[clap(default_value = "bash")]
+    shell: String,
+}
+
+impl ShellCompletions {
+    pub fn print(&self) -> anyhow::Result<()> {
+        clap_complete::generate(
+            Shell::from_str(&self.shell).map_err(|e| anyhow!("Shell {e}"))?,
+            &mut super::CliArgs::command(),
+            "release_plz",
```

Comment:
Not sure what is the best way to fetch exe name in Rust.

Argv[0] could be used, but then we need to forward it here via function argument possibly.

Alternative would be to setup `basename` function. 
Something like:

```rust
impl ShellCompletions {
    pub fn print(&self) -> anyhow::Result<()> {
        clap_complete::generate(
            Shell::from_str(&self.shell).map_err(|e| anyhow!("Shell {e}"))?,
            &mut super::CliArgs::command(),
            basename(std::env::current_exe()?.to_str().unwrap(), '/'),
            &mut std::io::stdout(),
            );
        Ok(())
    }
}

fn basename(path: &str, sep: char) -> &str {
    match path.rsplit(sep).next() {
        Some(p) => p.into(),
        None => path.into(),
    }
}
```
But that solution wouldn't really work on Windows.

Or we can simply keep "release_plz" hardcoded here and live with it.

Reply:
let's hardcode the binary name, it's totally fine!